src/terminal/mod.rs

```diff
@@ -128,11 +128,15 @@ impl TerminalCapabilities {
     }
 
     /// Terminal capabilities of Kitty.
-    pub fn kitty() -> TerminalCapabilities {
+    pub fn kitty(version: (u8, u8, u8)) -> TerminalCapabilities {
```

Comment:
I'd like for these constructors to always return the same set of capabilities.

Would you mind to leave `pub fn kitty()` as is, and add a separate and independent `pub fn kitty_with_links()`?  And the move the condition either into a separate `pub fn kitty_autodetect(version)` or even just to `detect()`?